src/components/filter.jsx

```diff
@@ -1,59 +1,71 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
-import { Dropdown, Form, Button, Row, Col } from 'bootstrap-4-react';
+import './style.css';
+import { Dropdown, Button } from 'bootstrap-4-react';
 
 class filter extends Component {
   state = {
-    content: '',
+    pin: '',
+    name: '',
   };
 
-  handleChange = e => {
+  clearfield() {
+    this.setState({ pin: '' });
+    this.setState({ name: '' });
+  }
+
+  handleNameChange = e => {
+    this.setState({
+      name: e.target.value,
+    });
+  };
+
+  handlePinChange = e => {
     this.setState({
-      content: e.target.value,
+      pin: e.target.value,
     });
   };
 
-  handleSubmit = e => {
+  handleSubmitName = e => {
     e.preventDefault();
-    this.props.pin(this.state.content);
-    this.setState({ content: '' });
+    this.props.username(this.state.name);
+  };
+
+  handleSubmitPIN = e => {
+    e.preventDefault();
+    this.props.pin(this.state.pin);
   };
 
   render() {
     return (
       <div>
-        <div
-          style={{
-            // marginTop: "2em",
-            textAlign: 'center',
-            display: 'flex',
-            alignItems: 'center',
-            justifyContent: 'center',
-          }}
-        >
-          <Form inline my='2 lg-0' onSubmit={this.handleSubmit}>
-            <Row>
-              <Col>
-                <Form.Input
-                  type='number'
-                  onChange={this.handleChange}
-                  value={this.state.content}
-                  placeholder='Filter by Pin-Code'
-                  mr='sm-8'
-                  min='111111'
-                  max='999999'
-                  style={{ width: '170%' }}
-                />
-              </Col>
-              <Col>
-                <Button outline success type='submit' mr='sm-0' my='sm-0'>
-                  Search
-                </Button>
-              </Col>
-            </Row>
-          </Form>
+        <div>
+          <form onSubmit={this.handleSubmitPIN} className='form1'>
+            <input
+              className='textbox'
+              type='number'
+              onChange={this.handlePinChange}
+              value={this.state.pin}
+              placeholder='Search by Pin-Code'
+              mr='sm-9'
+              min='111111'
+              max='999999'
+            />
+            <input title='Search' value='' type='submit' className='button' />
```

Comment:
What is assigned to `value` ?

Reply:
This value is used to display the search symbol.